### ISSUE TITLE
can't find or open a directory on a standalone build

### ISSUE THREAD

**Issue by vonflyhighace2** [godotengine/godot#15801]
**Godot version:**
Godot RC1

**OS/device including version:**
Windows 10 x64

**Issue description:**
locating / opening a directory works in the editor but once you create a standalone build the engine seems to not know where the directory is. finding and loading a specific file in said directory works but 
if you can find a file why not the directory the file is located in?

**Steps to reproduce:**
using something like directory_exists( path-to_directory) returns false on a standalone build, while it will work in the editor.

**Minimal reproduction project:**

**Comment by akien-mga** [comment]
That's by design, the default file access for exporter projects is the project folder (res://) and the user data dir (user://).

You can use FileDialog with ACCESS_FILESYSTEM if you need users to load/save files from their filesystem: http://docs.godotengine.org/en/latest/classes/class_filedialog.html?highlight=ACCESS_FILESYSTEM

**Comment by RabbitB** [comment]
@akien-mga I have the same issue (also RC1, Win 10 x64) with opening a directory in an exported project, but I'm trying to read from `res://` and not an external path.

I'll post the sample code that works in-editor but not on the exported build, but essentially I'm using it to discover every resource of a type in a folder, and load them, as there's a ton and manually adding a load statement for each resource is impractical and prone to missing one.

```
	var loaded_resources = []
	var directory = Directory.new()

	directory.open("res://sample/dir")
	directory.list_dir_begin()

	var file_name = directory.get_next()

	while (file_name != ""):

		if (directory.current_is_dir() || !file_name.ends_with(".wav")):

			file_name = directory.get_next()
			continue

		else:

			loaded_resources.append(load("res://sample/dir/" + file_name))
			file_name = directory.get_next()

**Comment by vonflyhighace2** [comment]
It seems the problem is with the .pck file. It has limited file access so folder access is not permitted but access to files are.

**Comment by vnen** [comment]
@RabbitB your issue is the same as #14562, essentially the files you're looking for are not there in an exported project because all the resources go to the `.import` folder.

**Comment by RabbitB** [comment]
@vnen Is the directory structure not also maintained? It never even gets to try and load the resources, because the directory can't be found.

**Comment by vonflyhighace2** [comment]
This Is a documentation issue. It should be noted that the directory class won't hold true on a exported .pck file, and work around are needed to ensure that the requested source material is read by other means once exported.

**Comment by vnen** [comment]
Still think this is a bug, I'm pretty sure the directory structure was supposed to be maintained within the PCK. The core implements DirAccess and FileAccess classes for the packed file.

**Comment by Phoenix1747** [comment]
So how is the status of this issue? Have encountered this issue on my own and not really found any workaround. #17607 is supposed to have fixed this issue, but is it also fixing #14562?
